ports/libbacktrace/portfile.cmake

```
@@ -0,0 +1,19 @@
+if (CMAKE_C_COMPILER_ID STREQUAL "MSVC")
+    vcpkg_fail_port_install(ALWAYS MESSAGE "The library relies on the C++ unwind API defined at https://itanium-cxx-abi.github.io/cxx-abi/abi-eh.html This API is only provided by GCC and clang. You are using MSVC.")
+endif()
```

This should be changed to be a warning instead of a fatal error -- additionally the `CMAKE_C_COMPILER_ID` is not available in `portfile.cmake` so this condition can currently never be triggered.

Instead, I would suggest:
```suggestion
if(VCPKG_TARGET_IS_WINDOWS)
    message("libbacktrace cannot be built using MSVC on Windows due to relying on the C++ unwind API https://itanium-cxx-abi.github.io/cxx-abi/abi-eh.html")
endif()
```